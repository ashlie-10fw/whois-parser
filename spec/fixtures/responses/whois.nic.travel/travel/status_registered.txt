Domain Name: travel.travel
Registry Domain ID: ff0390f7114c4ad99ac46ae56c00300f-DONUTS
Registrar WHOIS Server: whois.identitydigital.services
Registrar URL: https://identity.digital
Updated Date: 2022-11-18T00:00:14Z
Creation Date: 2005-10-04T21:44:27Z
Registry Expiry Date: 2023-10-03T23:59:59Z
Registrar: Donuts Internal (9998)
Registrar IANA ID: 9998
Registrar Abuse Contact Email: 
Registrar Abuse Contact Phone: 
Domain Status: ok https://icann.org/epp#ok
Registry Registrant ID: REDACTED FOR PRIVACY
Registrant Name: REDACTED FOR PRIVACY
Registrant Organization: Donuts Inc.
Registrant Street: REDACTED FOR PRIVACY
Registrant City: REDACTED FOR PRIVACY
Registrant State/Province: WA
Registrant Postal Code: REDACTED FOR PRIVACY
Registrant Country: us
Registrant Phone: REDACTED FOR PRIVACY
Registrant Phone Ext: REDACTED FOR PRIVACY
Registrant Fax: REDACTED FOR PRIVACY
Registrant Fax Ext: REDACTED FOR PRIVACY
Registrant Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Admin ID: REDACTED FOR PRIVACY
Admin Name: REDACTED FOR PRIVACY
Admin Organization: REDACTED FOR PRIVACY
Admin Street: REDACTED FOR PRIVACY
Admin City: REDACTED FOR PRIVACY
Admin State/Province: REDACTED FOR PRIVACY
Admin Postal Code: REDACTED FOR PRIVACY
Admin Country: REDACTED FOR PRIVACY
Admin Phone: REDACTED FOR PRIVACY
Admin Phone Ext: REDACTED FOR PRIVACY
Admin Fax: REDACTED FOR PRIVACY
Admin Fax Ext: REDACTED FOR PRIVACY
Admin Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Registry Tech ID: REDACTED FOR PRIVACY
Tech Name: REDACTED FOR PRIVACY
Tech Organization: REDACTED FOR PRIVACY
Tech Street: REDACTED FOR PRIVACY
Tech City: REDACTED FOR PRIVACY
Tech State/Province: REDACTED FOR PRIVACY
Tech Postal Code: REDACTED FOR PRIVACY
Tech Country: REDACTED FOR PRIVACY
Tech Phone: REDACTED FOR PRIVACY
Tech Phone Ext: REDACTED FOR PRIVACY
Tech Fax: REDACTED FOR PRIVACY
Tech Fax Ext: REDACTED FOR PRIVACY
Tech Email: Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
Name Server: ns-1812.awsdns-34.co.uk
Name Server: ns-458.awsdns-57.com
Name Server: ns-935.awsdns-52.net
Name Server: ns-1480.awsdns-57.org
DNSSEC: unsigned
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2023-03-14T15:28:47Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Terms of Use: Identity Digital Inc. provides this Whois service for information purposes, and to assist persons in obtaining information about or related to a domain name registration record. Identity Digital does not guarantee its accuracy. Users accessing the Identity Digital Whois service agree to use the data only for lawful purposes, and under no circumstances may this data be used to: a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the registrar's own existing customers and b) enable high volume, automated, electronic processes that send queries or data to the systems of Identity Digital or any ICANN-accredited registrar, except as reasonably necessary to register domain names or modify existing registrations. When using the Identity Digital Whois service, please consider the following: The Whois service is not a replacement for standard EPP commands to the SRS service. Whois is not considered authoritative for registered domain objects. The Whois service may be scheduled for downtime during production or OT&E maintenance periods. Queries to the Whois services are throttled. If too many queries are received from a single IP address within a specified time, the service will begin to reject further queries for a period of time to prevent disruption of Whois service access. Abuse of the Whois system through data mining is mitigated by detecting and limiting bulk query access from single sources. Where applicable, the presence of a [Non-Public Data] tag indicates that such data is not made publicly available due to applicable data privacy laws or requirements. Should you wish to contact the registrant, please refer to the Whois records available through the registrar URL listed above. Access to non-public data may be provided, upon request, where it can be reasonably confirmed that the requester holds a specific legitimate interest and a 
proper legal basis for accessing the withheld data. Access to this data can be requested by submitting a request via the form found at https://www.identity.digital/about/policies/whois-layered-access/ Identity Digital Inc. reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
