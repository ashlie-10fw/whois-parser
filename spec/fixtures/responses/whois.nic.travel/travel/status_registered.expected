#domain
  %s == "travel.travel"

#domain_id
  %s == "ff0390f7114c4ad99ac46ae56c00300f-DONUTS"


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2005-10-04 21:44:27 GMT}

#updated_on
  %s %CLASS{time}
  %s %TIME{2022-11-18 00:00:14 GMT}

#expires_on
  %s %CLASS{time}
  %s %TIME{2023-10-03 23:59:59 GMT}


#registrar
  %s %CLASS{registrar}
  %s.id           == "9998"
  %s.name         == "Donuts Internal (9998)"
  %s.url          == "https://identity.digital"

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "REDACTED FOR PRIVACY"
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "Donuts Inc."
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "WA"
  %s[0].country_code  == "us"
  %s[0].phone         == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "REDACTED FOR PRIVACY"
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "REDACTED FOR PRIVACY"
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "REDACTED FOR PRIVACY"
  %s[0].country_code  == "REDACTED FOR PRIVACY"
  %s[0].phone         == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "REDACTED FOR PRIVACY"
  %s[0].name          == "REDACTED FOR PRIVACY"
  %s[0].organization  == "REDACTED FOR PRIVACY"
  %s[0].address       == "REDACTED FOR PRIVACY"
  %s[0].city          == "REDACTED FOR PRIVACY"
  %s[0].zip           == "REDACTED FOR PRIVACY"
  %s[0].state         == "REDACTED FOR PRIVACY"
  %s[0].country_code  == "REDACTED FOR PRIVACY"
  %s[0].phone         == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].fax           == "REDACTED FOR PRIVACY ext: REDACTED FOR PRIVACY"
  %s[0].email         == "Please query the RDDS service of the Registrar of Record identified in this output for information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."


#nameservers
  %s %CLASS{array}
  %s %SIZE{4}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns-1812.awsdns-34.co.uk"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns-458.awsdns-57.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns-935.awsdns-52.net"
  %s[3] %CLASS{nameserver}
  %s[3].name == "ns-1480.awsdns-57.org"
